clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -1853,18 +1853,26 @@ static std::string getMangledNameImpl(CodeGenModule &CGM, GlobalDecl GD,
         break;
       case MultiVersionKind::Target: {
         auto *Attr = FD->getAttr<TargetAttr>();
+        assert(Attr &&
+               "Expected TargetAttr to be present for attribute mangling");
         const ABIInfo &Info = CGM.getTargetCodeGenInfo().getABIInfo();
         Info.appendAttributeMangling(Attr, Out);
         break;
       }
       case MultiVersionKind::TargetVersion: {
         auto *Attr = FD->getAttr<TargetVersionAttr>();
+        assert(
+            Attr &&
+            "Expected TargetVersionAttr to be present for attribute mangling");
```

Comment:
Please split the string in two so the formatting is less awkward.  Otherwise LGTM

Reply:
Thank you @efriedma-quic for reviews!

>>Please split the string in two so the formatting is less awkward. 

Done